### DISCUSSION TITLE
Mountain Duck not showing mount points on desktop or Location section of Finder side bar

### DISCUSSION THREAD

**Discussion by apastuszak** [iterate-ch/cyberduck#15703] [Q&A] [answered]
I upgraded to 4.15.4 to fix the "/" issue and now none of the connections show on my desktop, nor in the Locations section of the Finder window. I need to open them from either the menu bar app or create a shortcut for them under favorites.

**Comment by dkocher** [comment, marked as answer]
This is part of a workaround introduced in version 4.15.4 for the problem with mounts displaying as "/" or ":" respectively in macOS 14.4 discussed in #15698. You can revert the display of volumes on the desktop and the _Locations_ section in the sidebar with the [hidden configuration option](https://docs.mountainduck.io/mountainduck/preferences/#hidden-configuration-options)

```
fs.nfs.mount.nobrowse=false
```

However, be aware that because of the bug introduced in macOS 4.15.4 (`FB13675056`) the volume will always display as `:` in the _Network_ section of Finder.app.

**Comment by Jerry080801** [comment]
TYVM It worked

**Comment by oscu0** [comment]
No longer appears to work. Did I set it correctly with `defaults write io.mountainduck fs.nfs.mount.nobrowse false`?

**Reply by dkocher** [reply]
This is the default unless you run macOS 14.4.x.

**Comment by apastuszak** [comment]
Is this even needed anymore with Mountain Duck 5?

**Reply by dkocher** [reply]
It does not apply to connections using _Integrated_ mode.

**Comment by oscu0** [comment]
I was wrong, disks do appear on the desktop, I must have failed to notice for a while. although I did set the flag; I don't know if they do without it. Their names in Finder are correct.
